--- amrnb-0.0.1/typedef.h.orig	2004-03-05 00:00:02.000000000 +0100
+++ amrnb-0.0.1/typedef.h	2006-01-15 21:42:16.420609000 +0100
@@ -11,11 +11,13 @@
 
 #ifndef _TYPEDEF_H
 #define _TYPEDEF_H
-
-typedef char Word8;
-typedef unsigned char UWord8;
-typedef short Word16;
-typedef long Word32;
+
+#include <inttypes.h>
+
+typedef int8_t Word8;
+typedef uint8_t UWord8;
+typedef int16_t Word16;
+typedef int32_t Word32;
 typedef float Float32;
 typedef double Float64;
 
